src/ZODB/FileStorage/tests.py

```diff
@@ -185,6 +185,125 @@ def _save_index():
 
     """
 
+def pack_disk_full_copyToPacktime():
+    """Recover from a disk full situation by removing the `.pack` file
+
+`copyToPacktime` fails
+
+Add some data
+
+    >>> fs = ZODB.FileStorage.FileStorage('data.fs')
+    >>> db = ZODB.DB(fs)
+    >>> conn = db.open()
+    >>> conn.root()[1] = u'foobar'
```

Comment:
For Py3k compatibliity, it would be better to just leave it a native `str`:

- Python 3.2 doesn't allow unicode literals.
- The repr in the example below won't break.